packages/tentales/src/middleware/default-hooks.ts

```diff
@@ -20,6 +20,11 @@ const getProtectServiceRoutesHook = ({
 
   if (secret) {
     return ["protectServiceRoutesMiddleware", [protectServiceRoutesMiddleware]]
+  } else if (process.env.NODE_ENV === "production") {
+    log.error(
+      "Trying to run without authentication in production mode. You need to ensure that SERVER_SECRET is set in your environment.",
+    )
+    process.exit(1)
```

Comment:
I don't think we want to force this. A `log.warn` should suffice. 

It's not actually an error, and it's not leaking any data or anything. When it comes to updating data, we'll have to implement a user rights system anyways.

Reply:
Wasn't this what we decided on [here](https://github.com/iZettle/tentales/pull/9#discussion_r172499132)? 😋

Reply:
Oh... I didn't read the "fail" part. I'm all for it... except for failing ;)

Reply:
😂 Fine 

Though, I don't think we need a separate warning than the other one so I'll just revert the commit adding the exit statement. :)

Reply:
:bow: